### PR TITLE
Rebalance spawn rate of suits of armor in mansions

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
```SUMMARY: Balance "Suits of armor now have a chance to spawn fake or inferior weaponry."```  

#### Purpose of change
Right now, if you're a bog-standard survivor starting out, the first order of business is to go to a mansion. There you've got a pretty good chance (about 12%) to find a suit of armor with a decent medieval weapon. In particular, the awl pike, one of the game's best polearms, is rather likely to spawn. Combine this with the fact that mansions are often stocked full of food and drink, and they make a pretty good base of operations.

#### Describe the solution
I've created fake and inferior variants of every weapon that a mansion can spawn (that didn't already have one), and redid the numbers on the itemgroups so that you have a pretty good chance to see every variation of mannequins allowed. The proportions of weapons in each variant has remained untouched, however, and mannequins now have an even chance to spawn fake, inferior, or real varieties of each weapon.

#### Describe alternatives you've considered
Making suits of armor less common overall would feel a little odd. It's sort of a decoration a mansion would have from time to time, you know?

#### Additional context
These are the new items added by this PR:
[New items.txt](https://github.com/CleverRaven/Cataclysm-DDA/files/2381267/New.items.txt)

### CODE REVIEW

**Comment by Inglonias** [comment]
I would like to ask for some comment regarding the balancing of the fake and inferior weaponry. This is my first addition to the game in terms of items, so I'm open to raising or lowering their numbers.

**Comment by MT-Arnoldussen** [comment]
Currently, fake (aluminum) weapons are bad, but still somewhat useful if you really don't have anything else.

This might make sense for swords, but aluminum blunt weapons should really not be good for anything. I'd suggest changing their blunt damage to 1 or 2.

Might be a good idea to have them not have the Hammering quality either. I'm not sure an aluminum mace would make a very good hammer.

File: data/json/items/melee.json [hunk 1]
```diff
@@ -2007,5 +2063,159 @@
...
+    "looks_like": "pike",
+    "techniques": [ "IMPALE", "WBLOCK_2" ],
+    "qualities": [ [ "COOK", 1 ] ],
+    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "mace_fake",
+    "name": "mace",
+    "description": "A light, cheaply made replica of a medieval weapon consisting of a wood handle with a heavy iron end.",
```
Comment (MT-Arnoldussen) [inline data/json/items/melee.json]: It says it has an iron end, yet is made of aluminum?

File: data/json/items/melee.json [hunk 2]
```diff
@@ -2007,5 +2063,159 @@
...
+    "volume": 5,
+    "bashing": 22,
+    "price": 10000,
+    "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "morningstar_fake",
+    "name": "morningstar",
+    "description": "A light, cheaply made replica of a medieval weapon consisting of a wood handle with a heavy, spiked iron ball on the end.",
```
Comment (MT-Arnoldussen) [inline data/json/items/melee.json]: iron or aluminum?

Comment (Inglonias) [reply data/json/items/melee.json]: It says it's "a replica of" an iron weapon. This replica happens to be made of aluminum. Unclear? Yes. Technically correct English...? I think so.

File: data/json/items/melee.json (hunk 1 above)
Comment (Inglonias) [reply data/json/items/melee.json]: It says it's "a replica of" an iron weapon. This replica happens to be made of aluminum. Unclear? Yes. Technically correct English...? I think so.

Comment (Inglonias) [reply data/json/items/melee.json]: This description has been made clearer

File: data/json/items/melee.json (hunk 2 above)
Comment (Inglonias) [reply data/json/items/melee.json]: This description has been made clearer

**Comment by cosmiccoincidence** [comment]
You could add some of these items to pawn shops and other locations too maybe.

File: data/json/item_groups.json [hunk 3]
```diff
@@ -8638,7 +8638,11 @@
       ["battleaxe_fake", 6],
       ["battleaxe_inferior", 10],
       ["mace", 1],
+      ["mace_fake", 2],
+      ["mace_inferior", 3],
       ["morningstar", 2],
+      ["morningstar_fake", 3],
+      ["morningsta_inferiorr", 4],
```
Comment (Night-Pryanik) [inline data/json/item_groups.json]: morningstar_inferior

**Comment by WizardOfOoo** [comment]
I think those mace/morning star nerfs are harsh. Look at the other bashing damage for some other poor weapons.  Maybe a damage of 8 would be more appropriate.

Only bashing damage is listed.

digging stick: 8
Banjo: 7
burnt out torch: 8
rock in a sock: 8
pro fishing rod: 8
cudgel: 8

**Comment by Mecares** [comment]
Maybe the not made for combat of the fake weapons could be represented by the fragile flag in certain cases where the items are just made for decorative purposes?

**Comment by Inglonias** [comment]
I considered FRAGILE_MELEE, but none of the other fake weapons have it and I feel that nerfing every fake weapon is beyond the scope of this PR.

I also hate that flag with an irrational and fiery passion.

**Comment by Inglonias** [comment]
Jenkins rebuild

**Comment by Regilith** [comment]
I think you may need to add a comma after Jenkins.

**Comment by MxTurner** [comment]
Maybe fix the damage on the mace while you are at it? 2 is way too low.

**Comment by JacobKessler** [comment]
Now that mansions are JSON-ized, why not address this by making mansion variants that don't spawn medieval weaponry? It seems like rich folks who wanted suits of armor around as decoration wouldn't skimp on cheap versions, but I could easily see other styles of mansion that didn't have any medieval arms or armor.

**Pull request merged**